doxia-core/src/main/java/org/apache/maven/doxia/sink/impl/PipelineSink.java

```diff
@@ -28,8 +28,10 @@
 
 /**
  * May be used to invoke the same method on a List of Sinks.
+ * Consider using the more flexible {@link SinkWrapper} approach which doesn't require the use of dynamic proxies.
```

Comment:
Should we deprecate in favour of `SinkWrapper`?

Reply:
If you consider it better suited, I would trust your judgement here.

Reply:
Deprecated in https://github.com/apache/maven-doxia/pull/184/commits/2695cb63c7b638f5a01c961f320c5a785eb192b3.